Traceback (most recent call last):
  File "exploreByNext.py", line 74, in <module>
    lib.setSessionEndTime(setup['id'],setup["startedAtTime"])
  File "D:\Benny\università\Tesi\lib.py", line 249, in setSessionEndTime
    cursor.execute("update setupsessione set satus=%s where status !=%s  and id=%s",["ongoing","completed",setupId])
  File "C:\Users\Benny\AppData\Local\Programs\Python\Python38-32\lib\site-packages\mysql\connector\cursor.py", line 551, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\Benny\AppData\Local\Programs\Python\Python38-32\lib\site-packages\mysql\connector\connection.py", line 490, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\Benny\AppData\Local\Programs\Python\Python38-32\lib\site-packages\mysql\connector\connection.py", line 395, in _handle_result
    raise errors.get_exception(packet)
mysql.connector.errors.ProgrammingError: 1054 (42S22): Unknown column 'satus' in 'field list'
